package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint \"src/**/*.{ts,tsx,js,jsx}\" --fix",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
```

Comment:
this should be the right 'lint' script
 "lint": "eslint \"src/**/*.{ts,tsx,js,jsx}\" --fix"

Reply:
Done